lms/djangoapps/course_api/blocks/api.py

```diff
@@ -61,6 +62,18 @@ def get_blocks(
     # transform
     blocks = get_course_blocks(user, usage_key, transformers)
 
+    # filter blocks by types
+    if block_types and len(block_types) == 0:
```

Comment:
I don't know what type `block_types` will be, but empty containers in Python are false, so this expression is likely never going to be true.